bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_1j_Gpt-150To250_5f_NLO/GJets_1j_Gpt-150To250_5f_NLO_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
+
+define p = p b b~
+define j = j b b~
```

Comment:
please remove those lines, the p/j definition already include b quarks if *_no_b_mass model is imported